prep-html-skeleton/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+</html>
+<title>HTML Skeleton Exercise</title>
```

Comment:
The `title` element is an element that is meant to be in the `head` element as a child. Right now, this is in neither the `head` or `body` element, which is not allowed in HTML.